### PR TITLE
first sql script

### PR DESCRIPTION
**Pull request by evanrich2404**
Check me out, got it working and looking good on my local machine (looking like the examples given at least) and not throwing any obvious errors.

### CODE REVIEW

**Comment by Oggal** [comment]
Looks good to me, Do we want to add the second SQL script in this PR or open a separate one for it?

**Review by Oggal — APPROVED** [review]

File: setup_mysql_dev.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+-- preparing the database hbnb_dev for development
+CREATE DATABASE IF NOT EXISTS hbnb_dev;
+CREATE USER IF NOT EXISTS hbnb_dev@localhost IDENTIFIED BY 'hbnb_dev_pwd';
+GRANT ALL PRIVILEGES ON hbnb_dev.* TO 'hbnb_dev'@'localhost';
```
Comment (Oggal) [inline setup_mysql_dev.sql:4]: Database name needs corrected

File: setup_mysql_dev.sql [hunk 2]
```diff
@@ -0,0 +1,6 @@
+-- preparing the database hbnb_dev for development
+CREATE DATABASE IF NOT EXISTS hbnb_dev;
```
Comment (Oggal) [inline setup_mysql_dev.sql:2]: Does this database name need corrected as well?

**Review by Oggal — CHANGES_REQUESTED** [review]

**Review by Oggal — APPROVED** [review]
Looks great with the db corrected!

**Comment by evanrich2404** [comment]
Fixed the issues you spotted

**Pull request merged**